app/legacy_lib/mailchimp.rb

```diff
@@ -195,6 +195,12 @@ def self.hard_sync_lists(nonprofit, delete_from_mailchimp=false)
     end
   end
 
+  def self.sync_nonprofit_users(drip_email_list)
+    DripEmailList.find_each do |np_user| 
+      MailchimpNonprofitUserAddJob(np_user).perform_now
```

Comment:
This run all of these jobs on the current thread. What we're looking for is to run them all on background threads. We need to queue them up using `#perform_later`